models/_sector/nft/trades/chains/base/nft_base_base_trades.sql

```diff
@@ -0,0 +1,53 @@
+{{ config(
+    schema = 'nft_base',
+    alias = 'base_trades',
+    materialized = 'view'
+    )
+}}
+
+
+{% set nft_models = [
+ ref('alienswap_base_base_trades')
+] %}
+
+with base_union as (
+SELECT * FROM  (
+{% for nft_model in nft_models %}
+    SELECT
+        blockchain,
+        project,
+        project_version,
+        cast(date_trunc('day', block_time) as date) as block_date,
+        cast(date_trunc('month', block_time) as date) as block_month,
+        block_time,
+        block_number,
+        tx_hash,
+        project_contract_address,
+        trade_category,                 --buy/sell/swap
+        trade_type,                     --primary/secondary
+        buyer,
+        seller,
+        nft_contract_address,
+        nft_token_id,
+        nft_amount,                -- always 1 for erc721
+        price_raw,
+        currency_contract,
+        platform_fee_amount_raw,
+        royalty_fee_amount_raw,
+        platform_fee_address,   -- optional
+        royalty_fee_address,    -- optional
+        sub_tx_trade_id,
+--        tx_from,              -- not yet available in the base event tables
+--        tx_to,                -- not yet available in the base event tables
```

Comment:
don't we have to supply some sort of default value for these fields, if not available at source, in order for downstream models to pull from? otherwise union cross-chain will fail, since other chains supply values?

Reply:
see: https://github.com/duneanalytics/spellbook/pull/5055/files#r1446493114